### ISSUE TITLE
Nvda pronounces password when protecting documents in word

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#9270]
### Steps to reproduce:
Open a document in Word 2016.
Open the Files tab, and then information.
Now, try to protect documents.
And finally, try to encrypt a document.
Wait for nvda to read all the text and start typing the password.
expected nvda should not read the typed characters.
What occurs, the nvda read the characters typed.
Now press tab and then shift + tab to focus on the password field.
Now nvda correctly identifies the password field and does not read the characters as you type.
Now press the left arrow to walk through the entered password characters.
Expected, nvda should announce asterisk *
What happens, nvda is muted when walking through the characters of the password.
### Actual behavior:
nvda announces the entered password characters, as soon as the password field gains focus.
When pressing tab and shift + tab to place focus on the password box, nvda does not announce password characters as * asterisk while walking character by character.
### Expected behavior:
As soon as the password box receives focus, nvda should announce asterisk when entering the characters.
When pressing tab and shift + tab and walking with arrows to read the characters of the password the nvda should announce * asterisk.
#### NVDA installed/portable/running from source:
install
#### NVDA version:
2018.4.1
#### Windows version:
10 17134
#### Name and version of other software in use when reproducing the issue:
office 16.0.4266.1001.
sintetizador one core Brasil Daniel
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
no
There are 2 log files:
The first when the password box gains focus when you open the dialog box.
the second when the password box receives focus by pressing tab and shift + tab.
[word_password_1.txt](https://github.com/nvaccess/nvda/files/2860481/word_password_1.txt)
[word_password_2.txt](https://github.com/nvaccess/nvda/files/2860482/word_password_2.txt)

**Comment by Adriani90** [comment]
@fernando-jose-silva could you please test with the last alpha version of NVDA if this issue is still occuring?

**Comment by fernando-jose-silva** [comment]
this is resolved.
